httpobs/database/schema.sql

```diff
@@ -24,6 +24,7 @@ CREATE TABLE IF NOT EXISTS scans (
   tests_quantity                      SMALLINT   NOT NULL,
   grade                               VARCHAR(2) NULL,
   score                               SMALLINT   NULL,
+  likelihood_indicator                VARCHAR(8) NULL,
```

Comment:
I like to add a comment to the bottom of the schema to have the ALTER command as well.